ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,405 @@
+                setupLayoutConstraints()
+            }
+        }
+    }
+    /// Set `customBackgroundColor` in order to set the background color when using the custom color theme
+    @objc open var customBackgroundColor: UIColor = Constants.defaultBackgroundColor {
+        didSet {
+            if customBackgroundColor != oldValue {
+                setupColors()
+            }
+        }
+    }
+    /// Set `customTitleColor` in order to set the title's text color when using the custom color theme
+    @objc open var customTitleColor: UIColor = Constants.defaultTitleColor {
+        didSet {
+            if customTitleColor != oldValue {
+                setupColors()
+            }
+        }
+    }
+    /// Set `customSubtitleColor` in order to set the subtitle's text color when using the custom color theme
+    @objc open var customSubtitleColor: UIColor = Constants.defaultSubtitleColor {
+        didSet {
+            if customSubtitleColor != oldValue {
+                setupColors()
+            }
+        }
+    }
+    /// Set `customIconTintColor` in order to set the icon's tint color when using the custom color theme
+    @objc open var customIconTintColor: UIColor = Constants.defaultIconTintColor {
+        didSet {
+            if customIconTintColor != oldValue {
+                setupColors()
+            }
+        }
+    }
+    /// Set `customBorderColor` in order to set the border's color when using the custom color theme
+    @objc open var customBorderColor: UIColor = Constants.defaultBorderColor {
+        didSet {
+            if customBorderColor != oldValue {
+                setupColors()
+            }
+        }
+    }
+    /// Set `customWidth` in order to set the width of the card
+    @objc open var customWidth: CGFloat {
+        didSet {
+            if customWidth != oldValue {
+                setupLayoutConstraints()
+            }
+        }
+    }
+
+    /// The style of the card; vertical or horizontal. Style determines the layout of the content
+    private var style: CardStyle = .horizontal
+
+    private var iconView: UIImageView
+
+    private let primaryLabel: UILabel = {
+        let primaryLabel = UILabel()
+        primaryLabel.translatesAutoresizingMaskIntoConstraints = false
+        primaryLabel.adjustsFontForContentSizeCategory = true
+        primaryLabel.font = TextStyle.subhead.font
+        primaryLabel.textColor = Colors.textPrimary
+        primaryLabel.textAlignment = .natural
+        return primaryLabel
+    }()
+
+    private let secondaryLabel: UILabel = {
+        let secondaryLabel = UILabel()
+        secondaryLabel.translatesAutoresizingMaskIntoConstraints = false
+        secondaryLabel.adjustsFontForContentSizeCategory = true
+        secondaryLabel.font = TextStyle.footnote.font
+        secondaryLabel.textColor = Colors.textSecondary
+        secondaryLabel.textAlignment = .natural
+        return secondaryLabel
+    }()
+
+    private var sizeCategoryIncrease: Bool = false
```

Comment:
comments?

Reply:
I see that you are setting it to true for trailcollectiondidchange but what do you need it for?

Reply:
It's gone now.